option.go

```diff
@@ -35,16 +35,19 @@ var (
 		WithPolicyErrRetryInterval("1m"),
 		WithPubkeyErrRetryInterval("1m"),
 		WithJwkErrRetryInterval("1m"),
-		WithATProcessorParams(NewATProcessorParam(true, "1h", "1h")),
+		WithATProcessorParam(NewATProcessorParam(true, true, "1h", "1h", false, nil)),
 		WithRTVerifyRoleToken(true),
 		WithRCVerifyRoleCert(true),
 	}
 )
 
 type ATProcessorParam struct {
-	verifyCertThumbprint bool
-	certBackdateDur      string
-	certOffsetDur        string
+	verifyOAuth2AccessToken bool
```

Comment:
```suggestion
	enable bool
```

`WithRTVerifyRoleToken` => `WithEnableRoleToken` + `WithDisableRoleToken`
`WithRCVerifyRoleCert ` => similar

Reply:
done

Fix to AccessTokenParam
15b72ca

Updaate option func name
c0d7a1b

Fix parameter name to enableRoleToken
85bf144

Fix parameter name to enableRoleCert
99f5cb7

Fix parameter name to enable
90d6c2d

Fix test variable name
a931185